### ISSUE TITLE
Very slow and lagy opening of window

### ISSUE THREAD

**Issue by Oppa-YA** [File-New-Project/EarTrumpet#1418]
### Summary

Hello, I like your app, but every time when i click on the icon to open main window to change the slider it opens notisable slowly with lags and stutters.
My laptop has i7 + 16GB RAM and latest updated Windows 10 22H2 so that's not an issue.
Can you please add the ability to disable opening animation? It would make app more usable. Thank you.

### Steps to reproduce

When i click on the icon to open main window to change the slider it opens notisable slowly with lags and stutters.

### EarTrumpet version

Latest

### Windows version

10 22H2 with all updates

**Comment by CharlieIXXXXXXXXX** [comment]
Yes, I am sure it is something about EarTrumpet. No matter I use Windows 11 Pro or Windows 10 LTSC the problem persists.
Maybe the pop-up animation got to be improved. I know it may be hard, but if the developers fix it I will be more than delighted.
The comparison between EarTrumpet pop-up and the native pop-up is striking.

**Comment by Oppa-YA** [comment]
> Yes, I am sure it is something about EarTrumpet. No matter I use Windows 11 Pro or Windows 10 LTSC the problem persists. Maybe the pop-up animation got to be improved. I know it may be hard, but if the developers fix it I will be more than delighted. The comparison between EarTrumpet pop-up and the native pop-up is striking.

Thank you for confirmation, now I know it's not only me having that problem.
I would be happy even for just build without animation, until it can be fixed and working smooth.

**Comment by riverar** [comment]
Can you try a dev build? We have a fix there that I realize hasn't gone out yet (my bad). Will put out a new production release tonight.

**Comment by Oppa-YA** [comment]
> Can you try a [dev build](https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller)? We have a fix there that I realize hasn't gone out yet (my bad). Will put out a new production release tonight.

Thank you. I tried that dev build, it still opens window with micro stutters (feels like 20-30 fps animation) but it's like 35% faster than old build and feel already better.
I would use this build for now.

**Comment by notsyncing12309** [comment]
I have the same problem regardless of the release version or the dev, but the dev is a little smoother, but still the animation is jerky. I think it's worth adding the feature of completely disabling popup animation

**Comment by Oppa-YA** [comment]
> I have the same problem regardless of the release version or the dev, but the dev is a little smoother, but still the animation is jerky. I think it's worth adding the feature of completely disabling popup animation

Yes, no animation option would be much better if smooth animation is not achievable.

**Comment by pseudoenigma** [comment]
Sorry for bumping this, I realize the last reply was a year ago but I believe that nothing has changed between then and now, so it doesn't warrant opening a new issue. I have mentioned this before in a review I made a few years ago, and I've patiently waited for a resolution so I figured I'd chime in now that enough time has passed. 

It feels like the frame rate is limited, as another user mentioned it's something like 20-30 fps. I don't know if that's the real issue or if it's a limitation of the software, but it does feel unnatural compared to the UWP apps that open gracefully. By comparison, it slides open in a very jerky/chugging way. I am not using onboard graphics, and I have a reasonably good dedicated GPU (AMD RX 6750 XT) so this issue seems entirely localized to the software itself rather than my system's resources or dedication thereof. 

It's understandably not a high priority issue but if this were solved it would make EarTrumpet feel seamless in the Windows experience. The slow animation feels like a reminder that it's 3rd party software. It's not an intrusive bug so I will continue to patiently wait, but I would love to see this addressed.

**Comment by Oppa-YA** [comment]
> Sorry for bumping this, I realize the last reply was a year ago but I believe that nothing has changed between then and now, so it doesn't warrant opening a new issue. I have mentioned this before in a review I made a few years ago, and I've patiently waited for a resolution so I figured I'd chime in now that enough time has passed.
> 
> It feels like the frame rate is limited, as another user mentioned it's something like 20-30 fps. I don't know if that's the real issue or if it's a limitation of the software, but it does feel unnatural compared to the UWP apps that open gracefully. By comparison, it slides open in a very jerky/chugging way. I am not using onboard graphics, and I have a reasonably good dedicated GPU (AMD RX 6750 XT) so this issue seems entirely localized to the software itself rather than my system's resources or dedication thereof.
> 
> It's understandably not a high priority issue but if this were solved it would make EarTrumpet feel seamless in the Windows experience. The slow animation feels like a reminder that it's 3rd party software. It's not an intrusive bug so I will continue to patiently wait, but I would love to see this addressed.

Can confirm, I still have that issue and I never seen such behavior from other UWP/usual software.

**Comment by riverar** [comment]
Right, this is ultimately a WPF limitation; it uses a DirectX 9-based compositor that is separate from the system one, there's a bunch of interop going on, etc. Non-DirectComposition apps will likely never be as smooth as a "UWP" app or native shell fly-out.
